cypress/test/scripts/tools/tools-object.cy.js

```diff
@@ -0,0 +1,114 @@
+
+import { createToolObject } from "../../../../scripts/tools/tools-object";
+import { convertTools } from "../../../../scripts/tools/tools-object";
+describe('createToolObject', () => {
+  const testToolFile = {
+    title: 'Test Tool',
+    description: 'This is a test tool',
+    links: {
+      repoUrl: 'https://github.com/testuser/test-repo'
+    },
+    filters: {
+      hasCommercial: true
+    }
+  };
```

Comment:
Why `testToolFile` doesn't consists of `language`, `category` and `technology` fields?